src/features/lightspeed/inlineSuggestions.ts

```diff
@@ -827,6 +824,26 @@ export async function inlineSuggestionReplaceMarker(position: vscode.Position) {
       );
     });
   }
+  console.log(
+    "[inline-suggestions] Inline Suggestion Marker Handler removing extra whitespace"
+  );
+
+  // Clear the line of extra whitespace after the suggestion
+  // that causes ansible-lint errors
+  const selection = editor.selection;
+  const lineText = editor.document.lineAt(selection.active.line).text;
+  if (/^\s*$/.test(lineText)) {
+    editor.edit((editBuilder) => {
+      editBuilder.delete(
```

Comment:
I think you want to add `await` in front of `editor.edit` as you modified line 818.

Reply:
Thanks Tami. This one I was thinking it didn't matter really when it completed since nothing depends on it, but for consistency and to avoid surprises later, yes, let me take a look at that. Thank you!

Reply:
Done. Thanks!